result.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:bmi_calculator/constants.dart';
+import 'package:flutter/material.dart';
+import 'reusable_card.dart';
+import 'constants.dart';
+import 'bottom_button.dart';
+
+class ResultPage extends StatelessWidget {
+  const ResultPage(
+      {required this.resultText,
+      required this.resultVal,
+      required this.interp});
```

Comment:
Kudos 👍 for using named parameters in the constructor! This makes the code more readable and understandable, and makes it easier to maintain in the future.